lib/vehicle.rb

```diff
@@ -22,4 +26,14 @@ def antique?
   def electric_vehicle?
     @engine == :ev
   end
+
+  def set_plate_type
```

Comment:
Since you have `set_plate_type` you don't need to make `@plate_type` an attr_accesor. You are changing it within the class, not outside the class, so you can do lines 32, 34 and 36 without needing the attr_accessor